tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -1459,6 +1459,36 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 			})
 		})
 	})
+
+	ginkgo.It("TiKV mount multiple pvc", func() {
+
+		cluster := newTidbClusterConfig(e2econfig.TestConfig, ns, "tidb-multiple-pvc-scale", "admin", utilimage.TiDBV3Version)
+		cluster.Resources["pd.replicas"] = "1"
+		cluster.Resources["tikv.replicas"] = "2"
+		cluster.Resources["tidb.replicas"] = "1"
+		cluster.Clustrer.Spec.TiKV.StorageVolumes = []v1alpha1.StorageVolume{
+			{
+				Name:        "wal",
+				StorageSize: "2Gi",
+				MountPath:   "/var/lib/wal",
+			},
+			{
+				Name:        "titan",
+				StorageSize: "2Gi",
+				MountPath:   "/var/lib/titan",
+			},
+		}
+		cluster.Clustrer.Spec.TiKV.Config.Set("rocksdb.wal-dir", "/var/lib/wal")
+		cluster.Clustrer.Spec.TiKV.Config.Set("titan.dirname", "/var/lib/titan")
+		e2elog.Logf("deploying tidb cluster [%s/%s]", cluster.Namespace, cluster.ClusterName)
+		oa.DeployTidbClusterOrDie(&cluster)
+		oa.CheckTidbClusterStatusOrDie(&cluster)
+
+		ginkgo.By("scale multiple pvc tidb cluster")
+		cluster.ScaleTiKV(1)
```

Comment:
tikv cannot be scaled in with less than 3 replicas, we can test scaling in from 4 to 3 and then scale out from 3 to 4.